src/DssSpell.sol

```diff
@@ -46,14 +57,95 @@ contract DssSpellAction is DssAction {
 
     uint256 constant MILLION = 10**6;
 
+    address constant UNIV2DAIUSDC_GEM   = 0xAE461cA67B15dc8dc81CE7615e0320dA1A9aB8D5;
+    address constant UNIV2DAIUSDC_JOIN  = 0xA81598667AC561986b70ae11bBE2dd5348ed4327;
+    address constant UNIV2DAIUSDC_FLIP  = 0x4a613f79a250D522DdB53904D87b8f442EA94496;
+    address constant UNIV2DAIUSDC_PIP   = 0x25CD858a00146961611b18441353603191f110A0;
```

Comment:
Is it OK that the OracleFactory is relied on the PIP?

    $ seth call $PIP 'wards(address)' 0x9f6d3a34b493b2e8b6bdb04e45dc640cdf691a40
    0x0000000000000000000000000000000000000000000000000000000000000001

Reply:
It would be better it denies itself like the other factories do (as a standard approach).
In practice it won't affect on anything, but we might want remove these authorities in a next spell.